pages/docs/environments/environment-variables.mdx

```diff
@@ -14,6 +14,7 @@ Environment variables are needed in the `.env` file to allow the application to
 | GITHUB_ID                        | abcedfghil12345                    | true     | GitHub OAuth ID                                     |
 | GITHUB_SECRET                    | a1b2c3e4f5e6                       | true     | GitHub OAuth Secret                                 |
 | NEXTAUTH_SECRET                  | afsfdsafadsf                       | true     | Next Auth Secret, this can be any random characters |
+| GITHUB_API_TOKEN                 | djsbjsjacdnk                       | true     | Github Personal Access Token                        |
```

Comment:
In the Zod config, it is marked as optional but in the docs it is marked as required?

I will update the docs to make it optional, as it should still work without it but will have a lower limit

```suggestion
| GITHUB_API_TOKEN                 | djsbjsjacdnk                       | false     | Github Personal Access Token                        |
```